client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,17 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
```

Comment:
There's only this one `render...()` method here, but let's be careful we don't end up with another component that looks like the theme sheet (tons of `render...()`s). One alternative to consider here would be to put this into a stateless component in the same file:

```js
const MultiSiteJetpackWarning = ( { translate } ) => (
	<EmptyContent
		title={ translate( 'Not available for multi site' ) }
		/* ... */
	/>
);
```

If you decide to stick with the method, can we at least rename it? `NotAvailable` is just so generic, while the error message is really quite specific for JP multisite...